### PR TITLE
support more csv comma

### PR DESCRIPTION
**Pull request by knightXun**
support more csv comma

we want to import ldbc test data, the csv comma is '|'

### CODE REVIEW

File: pkg/config/config.go [hunk 1]
```diff
@@ -86,6 +86,7 @@ type YAMLConfig struct {
 	Description          *string               `json:"description" yaml:"description"`
 	NebulaClientSettings *NebulaClientSettings `json:"clientSettings" yaml:"clientSettings"`
 	LogPath              *string               `json:"logPath" yaml:"logPath"`
+	CVSComma             *string               `json:"csvComma" yaml:"csvComma"`
```
Comment (yixinglu) [inline pkg/config/config.go]: This is a csv option, why not move into the `CSVConfig` struct ? And I think the option's name `Delimiter` is better than `CVSComma`.

File: pkg/csv/reader.go [hunk 2]
```diff
@@ -14,10 +14,12 @@ type CSVReader struct {
 	CSVConfig *config.CSVConfig
 	reader    *csv.Reader
 	lineNum   uint64
+	Comma     rune
```
Comment (yixinglu) [inline pkg/csv/reader.go]: If CSVConfig stored delimiter, this field would be needless.

File: pkg/reader/reader.go [hunk 3]
```diff
@@ -28,10 +28,10 @@ type FileReader struct {
 	StopFlag    bool
 }
 
-func New(fileIdx int, file *config.File, clientRequestChs []chan base.ClientRequest, errCh chan<- base.ErrData) (*FileReader, error) {
+func New(fileIdx int, comma rune, file *config.File, clientRequestChs []chan base.ClientRequest, errCh chan<- base.ErrData) (*FileReader, error) {
 	switch strings.ToLower(*file.Type) {
 	case "csv":
-		r := csv.CSVReader{CSVConfig: file.CSV}
+		r := csv.CSVReader{CSVConfig: file.CSV, Comma: comma}
```
Comment (yixinglu) [inline pkg/reader/reader.go]: ditto

File: pkg/config/config.go (hunk 1 above)
Comment (knightXun) [reply pkg/config/config.go]: you need think all csv file have one common format

File: pkg/csv/reader.go (hunk 2 above)
Comment (knightXun) [reply pkg/csv/reader.go]: you need think all csv file have one common format

File: pkg/reader/reader.go (hunk 3 above)
Comment (knightXun) [reply pkg/reader/reader.go]: you need think all csv file have one common format

**Comment by knightXun** [comment]
ping @yixinglu

File: pkg/config/config.go (hunk 1 above)
Comment (yixinglu) [reply pkg/config/config.go]: I don't think so. The meaning of designing CSVConfig is to let user configure each data file, and in the future, maybe there's some else XMLConfig or ExcelConfig to support excel or xml file. We shouldn't put the option of csv configuration in global scope.

File: pkg/csv/reader.go (hunk 2 above)
Comment (yixinglu) [reply pkg/csv/reader.go]: > you need think all csv file have one common format

Each file will have their owned csv reader, and the delimiter option has been configured in `CSVConfig`, why not to use it directly?

File: pkg/reader/reader.go (hunk 3 above)
Comment (yixinglu) [reply pkg/reader/reader.go]: > you need think all csv file have one common format

Please refer the above replies.

**Comment by yixinglu** [comment]
> ping @yixinglu

It's better to add some test examples for this PR.

File: pkg/config/config.go (hunk 1 above)
Comment (knightXun) [reply pkg/config/config.go]: This is done, Config CSV comma for each file.I removed it

**Comment by yixinglu** [comment]
@knightXun I have support the other delimiters for csv in #57 , please review it.

**Pull request closed without merging**